lib/aur-search

```diff
@@ -9,63 +9,51 @@ query_type=search
 search_by=name-desc
 sort_key=Name
 
-json_short() {
-    local Name Version NumVotes Maintainer OutOfDate Description
+parse_short() {
```

Comment:
unrelated name change

Reply:
Functions `json_short` and `json_long` are split in two stages - parsing PRC results and printing. This is parsing, thus named `parse.